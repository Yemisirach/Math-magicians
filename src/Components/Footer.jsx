import React from "react";

export default function Footer() {
  return (
    <div id="footer">
      <div>
        <h2>Math Magician project</h2>
        <h3>By:- Yemisirach Tamirat.</h3>
      </div>
      <div>
        Â© 2023 All rights reserved. Math Magicians. Privacy Policy | Terms of
        Use | Contact Us
      </div>
    </div>
  );
}
