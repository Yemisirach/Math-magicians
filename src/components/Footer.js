import React from 'react';

export default function Footer() {
  return (
    <div id="footer">
      <h2>Math Magician project</h2>
      <h3>By, Yemisirach Tamirat.</h3>
    </div>
  );
}
