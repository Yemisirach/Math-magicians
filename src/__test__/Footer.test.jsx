import React from "react";
import renderer from "react-test-renderer";
import Footer from "../components/Footer";

describe("Calculator component", () => {
  it("should render correctly and match the saved snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});