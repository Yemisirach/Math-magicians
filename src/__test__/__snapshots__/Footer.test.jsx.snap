// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component should render correctly and match the saved snapshot 1`] = `
<div
  id="footer"
>
  <div>
    <h2>
      Math Magician project
    </h2>
    <h3>
      By:- Yemisirach Tamirat.
    </h3>
  </div>
  <div>
    Â© 2023 All rights reserved. Math Magicians. Privacy Policy | Terms of Use | Contact Us
  </div>
</div>
`;
